contracts/factories/WrappedExternalBribeFactory.sol

```diff
@@ -4,14 +4,11 @@ pragma solidity 0.8.13;
 import {WrappedExternalBribe} from 'contracts/WrappedExternalBribe.sol';
 
 contract WrappedExternalBribeFactory {
-    address public immutable voter;
+    bool internal _initialized;
+    address public voter;
     mapping(address => address) public oldBribeToNew;
     address public last_bribe;
 
-    constructor(address _voter) {
-        voter = _voter;
-    }
-
     function createBribe(address existing_bribe) external returns (address) {
```

Comment:
[Suggestion] If we confirm we need to remove voter in constructor, please:
1. add a require statement to make sure it is initialized and voter is set.
2. add in deployment script to setVoter

Reply:
@t0rbik

Reply:
> [Suggestion] If we confirm we need to remove voter in constructor, please:
> 
> 1. add a require statement to make sure it is initialized and voter is set.
> 2. add in deployment script to setVoter -> already done here (https://github.com/Velocimeter/instruments/blob/tests/deploy/15_init.ts#L47)

Could you be more specific about first point?

Reply:
`createGauge` function of `Voter.sol` can be called before `voter` is set in `WrappedExternalBribeFactory`, so adding following require statement into `createBribe` function will help prevent it from happening.

```
require(_initialized, "Not initialized");
```